java/fury-benchmark/src/main/java/org/apache/fury/benchmark/Identity2IdMap.java

```diff
@@ -26,16 +26,16 @@
 public class Identity2IdMap {
   private static final int RESERVE = 4;
   private static final int MAX_DEPTH = 4;
-  static int[] prim = {
```

Comment:
This class is copied from fst, it's unnecessary to update it

Reply:
Done. revert all change in this class